spec/requests/api/v2/users_spec.rb

```diff
@@ -48,21 +48,121 @@
     it { should have_http_status(:success) }
 
     it 'has a data section with the correct attributes' do
-      json_response = JSON.parse response.body
-      expect(json_response).to eq(
-        "data" => {
-          "id" => user.id.to_s,
-          "type" => "users",
-          "attributes" => {
-            "first_name" => user.first_name,
-            "last_name" => user.last_name,
-            "email" => user.email
-          }
-        },
-        "jsonapi" => {
-          "version" => "1.0"
-        }
+      json_response = JSON.parse(response.body)
+      data_attributes = json_response['data']['attributes']
+      expect(data_attributes['email']).to eq(user.email)
+      expect(data_attributes['first_name']).to eq(user.first_name)
+      expect(data_attributes['last_name']).to eq(user.last_name)
+    end
+  end
+
+  describe 'PATCH update' do
+    let(:params) { {} }
+
+    def perform_request
+      Timecop.freeze
```

Comment:
I don't think we actually need this 🤔

Reply:
Maybe it's safe to leave it in, just in case though. we can change it to use the block syntax. `Timecop.freeze` always scares the crap out of me. :P